smart_pet_buddy/lib/controlpanel.dart

```diff
@@ -244,15 +247,19 @@ class ControlpanelState extends State<Controlpanel> {
             Flexible(
               flex: 1,
               child: Container(
-                child: TextButton(
-                  onPressed: _forward,
-                  child: Text("forward"),
-                  key: Key('forwards'),
-                  style: ButtonStyle(
-                      backgroundColor: isForward
-                          ? MaterialStateProperty.all(Colors.red)
-                          : MaterialStateProperty.all(Colors.blue),
-                      foregroundColor: MaterialStateProperty.all(Colors.white)),
+                height: 120,
+                child: Transform.rotate(
+                  angle: 270 * pi / 180,
+                  child: IconButton(
+                    key: Key('forwards'),
+                    onPressed: _forward,
+                    splashRadius: 1.0,
+                    iconSize: 90,
+                    color: isForward ? HexColor("809ec2") : HexColor("aebccb"),
```

Comment:
Nice that you introduced HexColor! It will very helpful when working more with the UI experience.